### DISCUSSION TITLE
Hooks & Helpers Coverage

### DISCUSSION THREAD

**Discussion by xindzju** [microsoft/ebpf-for-windows#3164] [General]
Do we have a timeline for adding more hooks and helpers for other subsystem/kernel modules instead of Network only?

**Comment by Alan-Jowett** [comment]
Can you please provide more details on what hooks you are interested in and the scenario for them? If there are specific hooks you need, please file an issue with details. 

As an example #3158 is tracking adding a hook for the filesystem.

If there is a milestone assigned, then that gives an indication of the expected timeline, but if it's marked backlog, then there is no timeline and contributions would be greatly appreciated.
